README.md

```diff
@@ -1,364 +1,459 @@
+#18 8.478   TodoApi -> /app/out/
+#18 DONE 8.6s
 
-
-
-If your container is meant to be consumed (ala base image) DockerHub is probably the best place to host your container.  It allows easy access to it and works seamlessly in the Dockerfile.  If your container is a private container then you need to know where they are running. Your containers should be registered in the same cloud you use for hosting them. 
+#11 [stage-1 2/3] WORKDIR /app
+#11 sha256:5790d9d246621300d2780afef58e515532a82e570b8580baf68a70f6b5e8a4cf
+#11 CACHED
+
+#19 [stage-1 3/3] COPY --from=build-env /app/out .
+#19 sha256:523fdcb93ec048adf029f85134458e7ead7e3a5146224cef071911ac449eff09
+#19 DONE 0.2s
+
+#20 exporting to image
+#20 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
+#20 exporting layers
+#20 exporting layers 0.2s done
+#20 writing image sha256:7e28663f832381c26e1ee7fc058446d2895aa1fe83c91212010859cf45c1bcf9 0.0s done
+#20 naming to docker.io/library/todov1 0.0s done
+#20 DONE 0.3s
+
+Use 'docker scan' to run Snyk tests against images to find vulnerabilities and learn how to fix them
+```
+
+Next we can run the container:
+
+```bash
+docker run -it --rm -p 5000:5000 -e "ASPNETCORE_URLS=http://+:5000" --name To_Do_App todov1
+```
+
+> **_NOTE:_** Bash users might get an error like "the input device is not a TTY. If you are using mintty, try prefixing
+> the command with 'winpty'." If this is the case, use a different terminal like PowerShell or use winpty to run the
+> command:
+>
+> ```bash
+> winpty docker run -it --rm -p 5000:5000 -e "ASPNETCORE_URLS=http://+:5000" --name To_Do_App todov1
+> ```
+>
+> If the above command fixes the error, you might want to create a Bash alias to handle this in the future without
+> typing 'winpty':
+>
+> ```bash
+> echo "alias docker='winpty docker'" >> ~/.bashrc
+> ```
+>
+> Now you should be able to run the original Docker command without any issues.
+
+Let's take a closer look at the arguments in the above command:
+
+- **-it** runs it interactively at the command prompt, as opposed to detached (i interactive t terminal)
+- **--rm** automatically removes the container at exit (to clean up your local environment)
+- **-p** sets up the port to connect your local port of 5000 to the port 5000 on the **container**
+- **-e** sets an environment variable (We don't really need this since we set it in **Program.cs**)
+- **--name** is the name of the container
+- and lastly, the name you gave it in the previous step.
+
+To prove to yourself that the container is running the app on your machine, open a browser and navigate to
+<http://localhost:5000/api/todos>.
+
+If you want to look around the container you can have it give you a Bash prompt when you run it. Normally you can just
+add /bin/bash to the end of the command but if you have an entrypoint defined (we do) you have to run the command like
+this:
+
+```bash
+docker run -it --rm -p 5000:5000 --name To_Do_App --entrypoint bash todov1
```

Comment:
New, previous command (`docker run -it --rm -p 5000:5000  --name ToDo_App --entrypoint /bin/bash todov1 `) didn't work. Name was wrong but also `--entrypoint /bin/bash` seemed out of date